vllm/engine/async_llm_engine.py

```diff
@@ -130,7 +130,10 @@ def process_request_output(self,
         """Process a request output from the engine."""
         request_id = request_output.request_id
 
-        self._request_streams[request_id].put(request_output)
+        # Avoid a KeyError which can occur if the request was aborted while the
+        # outputs were generated
+        if request_id in self._request_streams:
+            self._request_streams[request_id].put(request_output)
```

Comment:
nit: can avoid double lookup
```suggestion
        if (stream := self._request_streams.get(request_id)) is not None:
            stream.put(request_output)
```
same for the other one